firebase-firestore/ktx/src/test/kotlin/com/google/firebase/firestore/TestUtil.kt

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Google LLC
```

Comment:
Sync PRs to include the deletion of this code from the non test code.

Reply:
I have synced the PRs now, but these two are the test codes I need:
1. `documentReference` is a dummy docRef with firestore set to be `null`, this is needed for unit test this decoder, because decoder's constructor requires a not null `documentReference`;
2. `assertThrows` is the method to check throws with Truth, this is used for both integration test and unit test.